docs/content/observability/tracing/opentelemetry.md

```diff
@@ -120,6 +120,55 @@ tracing:
 --tracing.openTelemetry.path=/foo/v1/traces
 ```
 
+#### `globalTags`
```

Comment:
Shouldn't we talk about attributes here? (rather than tags)

Reply:
`globalTags` has been renamed to `globalAttributes`